core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2183,6 +2183,13 @@ object SparkContext extends Logging {
   }
 
   /**
+   * Return SparkContext
+   */
+  def get(): SparkContext = {
```

Comment:
No please avoid introducing new random APIs in an unrelated pull request.

Also it would be great to not have the lower level depend on an upper level (e.g. network rpc depending on sparkcontext).